### PR TITLE
Added usage documentation

### PR DESCRIPTION
**Pull request by nusje2000**

### CODE REVIEW

**Comment by nusje2000** [comment]
See #34

File: USAGE.md [hunk 1]
```diff
@@ -46,6 +46,53 @@ nusje2000_feature_toggle:
...
+        access_control:
+            - { path: '^/feature-1-protected', ips: [ 127.0.0.1 ], port: 8080, features: { feature_1: true } }
+            - { path: '^/feature-2-protected', ips: [ 127.0.0.1 ], features: { feature_2: true } }
+            - { path: '^/feature-3-protected', host: symfony\.com$, features: { feature_3: false } }
+            - { path: '^/feature-4-and-5-protected', methods: [ POST, PUT ], features: { feature_4: false, feature_5: true } }
+```
+
+##### Defining route requirements
+
+- **path**: a regex representing the path that should be protected
```
Comment (dbrekelmans) [inline USAGE.md:72]: Are you sure this is a regex? I thought it was a glob pattern actually, but I might be wrong

Comment (aeviiq) [reply USAGE.md:72]: It is a regular expression, without delimiters.

File: USAGE.md [hunk 2]
```diff
@@ -46,6 +46,53 @@ nusje2000_feature_toggle:
...
+
+#### Access control
+
+By using access control you can protect http routes from being access if certain requirements are not met.
+
+```yaml
+nusje2000_feature_toggle:
+    environment:
+        access_control:
+            - { path: '^/feature-1-protected', ips: [ 127.0.0.1 ], port: 8080, features: { feature_1: true } }
```
[the remark this answers is not in the document]
Comment (nusje2000) [reply USAGE.md:64]: I though I added this, but I think it got removed when rewriting some of the documentation. I will add it.

**Pull request merged**